### DISCUSSION TITLE
502 Bad Gateway

### DISCUSSION THREAD

**Discussion by SSP6904** [community/community#73462] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

Codespaces on the web is giving me a 502 bad gateway message.

It was last working yesterday night. Can someone look into this for me?

Thanks! 🙂

![image](https://github.com/community/community/assets/75172901/4613bc45-7ccd-47c0-a0b3-8ef8997ee80e)

**Comment by andreas-sqsh** [comment, marked as answer]
`killall node` and restart or create a new machine if nothing helps

sadly for me a known issue by know

**Comment by start-hobby** [comment]
I also have the same problem, but it came up after burglary Advmanager to upgrade in a root folder from my repository

**Comment by SSP6904** [comment]
I figured it was my internet acting up.. Sorry about that
